src/ServerPrivate.cc

```diff
@@ -59,7 +59,7 @@ struct LoggingPlugin
   public: static std::string &RecordPluginName()
   {
     static std::string recordPluginName =
-      "ignition::gazebo::systems::LogRecord";
+      "gz::sim::systems::LogRecord";
```

Comment:
Just noticed this while testing something else. This change in together with the logic in https://github.com/gazebosim/gz-sim/blob/59b76c2d827cd928d27139a2ecd94f2a9a8d4de2/src/ServerPrivate.cc#L214 means sdf fileds with `ignition::gazebo::systems::LogRecord` won't work in Garden. Can you confirm @methylDragon ?

Reply:
You're right actually :o
It might be better to do a suffix comparison/regex match for `LogRecord$` then :/